### ISSUE TITLE
Script Kiwix Install Appears to Conflict

### ISSUE THREAD

**Issue by j-schwartz** [rachelproject/rachelplus#54]
I started with USB v6 in mode 3 (total reformat).  I tried to then build a CAP content from the ground up.  My steps were "download english" "download KA-Lite" and then "download Wikipedia"

I thought I should go through kiwix install to get wikipedia, but doing so appears to have corrupted the old clean install:

![image](https://cloud.githubusercontent.com/assets/9270388/15486467/eca9714a-20fa-11e6-8f9c-01ed9044fbc5.png)

I subsequently tried to "update modules" 
![image](https://cloud.githubusercontent.com/assets/9270388/15486481/0b2f94be-20fb-11e6-80a3-3089dd4bed5a.png)

and then "add modules"

![image](https://cloud.githubusercontent.com/assets/9270388/15486485/1236fc16-20fb-11e6-94d2-0c4c0524cc2c.png)

**Comment by spatiald** [comment]
For the add module function...I am not sure what happened.  It looks like I
added something I shouldn't have.  I fixed the code and the latest
configure script should have resolved the add module errors.

I also fixed the kiwix install error issues.  Those errors are not related
to functionality but to the fact that there were no zim files in the
library.xml when kiwix was first installed.  The install function is two
parts; install and then repair library.  I stopped kiwix from starting
until after the repair library function.  I muted the errors found during
the repair library function as they are due to kiwix trying to install
invalid .zimaa, etc files...some are legit and some are secondary zim files
for the master one.
